.ci/install_linux_catkin.sh

```diff
@@ -6,29 +6,14 @@ cd "${HOME}/workspace"
 cp -r "${GITHUB_WORKSPACE}" src
 ./scripts/internal-distro.py --workspace=src distribution.yml --repository "${REPOSITORY}" ${REQUIRED_ONLY}
 
-if [ $BUILD_NAME = TRUSTY_FULL_DEBUG ]; then
-  sudo apt-get install -y clang-format-6.0
-fi
+# Catkin on Focal-only
+$SUDO apt-get install -y clang-format-10
 
 if [ "$BUILD_AIKIDOPY" = "ON" ]; then
   $SUDO apt-get -y install python3-dev python3-numpy
   $SUDO apt-get -y install python3-pip -y
   $SUDO pip3 install pytest -U
 
-  if [ $(lsb_release -sc) = "trusty" ] || [ $(lsb_release -sc) = "xenial" ] || [ $(lsb_release -sc) = "bionic" ]; then
```

Comment:
Same Q about whether we want to include if statements to maintain compatibility for earlier versions of Ubuntu or not